casper/src/main/resources/Pos.rhox

```diff
@@ -6,6 +6,7 @@
 // quarantineLength - the length of the quarantine time in blocks
 // numberOfActiveValidators - max number of active validator in a shard
 // posMultiSigPublicKeys - public keys
+// posVaultPubKey - public key for transfers from the PoS vault
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.